hcl/printer/printer.go

```diff
@@ -60,5 +60,8 @@ func Format(src []byte) ([]byte, error) {
 		return nil, err
 	}
 
+	// Add trailing newline to result
+	buf.WriteString("\n")
```

Comment:
I'm going to defer to @jen20 since he's spent more sadness in the arena than me, but I think this is going to break the printer on Windows :frowning:. I think we need to write `/r/n` or whatever their newline character is?